test/test.js

```diff
@@ -318,7 +319,31 @@ describe('sourcemap-concat', function() {
     });
   });
 
-  it('can ignore empty content', function() {
+  it('can build empty files with allowNone disabled', function() {
+    var node = concat(emptyFixture, {
+      outputFile: '/empty.js',
+      inputFiles: ['*.js']
+    });
+    builder = new broccoli.Builder(node);
+    return builder.build().then(function(result) {
+      expectFile('empty.js').in(result);
+      expectFile('empty.map').in(result);
+    });
+  });
+
+  it('can build empty files with allowNone disabled and no source-maps', function() {
+    var node = concat(emptyFixture, {
+      outputFile: '/empty-no-sourcemap.js',
```

Comment:
Looks like these tests might be what is creating the `test/expected/empty-no-sourcemap.js` output I mentioned above? We should make sure that running the tests doesn't result in a "dirty" repo...